test/ParseableInterface/ModuleCache/force-module-loading-mode-archs.swift

```diff
@@ -56,7 +56,7 @@
 // RUN: not %target-swift-frontend -typecheck -parse-stdlib -module-cache-path %t/MCP -I %t %s 2>&1 | %FileCheck -check-prefix=FROM-INTERFACE %s
 
 // 6. Both are present but the module can't be opened.
-// RUN: chmod a-r %t/Lib.swiftmodule/%target-swiftmodule-name
+// RUN: %{python} %S\..\Inputs\make-unreadable.py %t/Lib.swiftmodule/%target-swiftmodule-name
```

Comment:
Uh, you slipped backslashes in here!

Reply:
lol, thats what happens when you have been typing `\` for too long :-(.